app/views/partials/_postCard.html.erb

```diff
@@ -0,0 +1,24 @@
+<% (1..items).each_with_index do |index| %>
+  <div class="horizontal-panel flex flex-col border-black">
+    <div class="px-5">
+      <p class="text-md font-bold">Post #<%= index %></p>
+      <p class="text-lg">
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia...
+      </p>
+    </div>
+    <div class="px-5 flex justify-end -mt-30">
+      <p class="text-sm">Comments: x, Likes: y</p>
+    </div>
+  </div>
+
+  <%# post comments %>
+  <% if current_page == 'posts' %>
+    <%= render partial: "partials/commentCard", locals: { items: 5 } %>
+  <% end %>
+<% end %>
```

Comment:
- [ ] Following best practices the show view should be similar like:
```ruby
% @user.recent_post.each do |post| %>

<h4><%= link_to post.title, user_post_path(post) %></h4>
<p><%= post.text %></p>
<p>
<small>Comments: <%= post.comments_counter || 0 %>,</small> 
<small>Likes: <%= post.likes_counter || 0 %></small>
</p>

<% end %>

<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>
```

Reply:
This has been fixed. Thanks for picking it up.